app/templates/books/new.hbs

```diff
@@ -3,10 +3,15 @@
 <h2 class="textCenter">{{t 'book.new'}}</h2>
 <form>
   <div class="form-group">
-    {{input value=title placeholder=(t 'book.title') class="form-control" id="titleInput"}}
+    {{input value=title placeholder=(t 'book.title') class="form-control"}}
   </div>
   <div class="form-group">
-    {{input value=author placeholder=(t 'book.author') class="form-control" id="authorInput"}}
+    {{input value=author placeholder=(t 'book.author') class="form-control"}}
   </div>
+  <div class="form-group">
+    {{input value=cover placeholder=(t 'book.cover') class="form-control"}}
+  </div>
+  <img src={{cover}}>
```

Comment:
<img /> element should always contain alt (at least with empty string) attribute based on w3c requirements.